__tests__/app.test.js

```diff
@@ -41,9 +42,42 @@ describe("/api", () => {
       .get("/api")
       .expect(200)
       .then((response) => {
-        expect(response.body.docs).toHaveProperty("GET /api");
-        expect(response.body.docs).toHaveProperty("GET /api/topics");
-        expect(response.body.docs).toHaveProperty("GET /api/articles");
+        expect(response.body).toHaveProperty("endpoints");
+        expect(response.body.endpoints).toEqual(endpoints);
+      });
+  });
+});
+
+describe("/api/articles/:article_id", () => {
+  test("GET:200 retrieves an article by its ID", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
```

Comment:
All the articles will have these properties so you are not checking the endpoint works correctly. Instead, copy and paste the article to check it looks the same.